### PR TITLE
Problem list view feature

### PR DESCRIPTION
**Pull request by YannyTwoo**
# Descripción

Added a new item to the problems dropdown in NavbarItems.vue file that leads to "/problems" href while the pre existing one's name was changed.
All lang files have been adjusted too however need an approval on translations.

Before:
Screenshot 2023-04-03 at 4 59 17 AM
Now:
Screenshot 2023-04-03 at 4 58 25 AM

Fixes: #4996 

# Comentarios

Requesting reviewers to verify the translations.

# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [x] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [x] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

**Comment by YannyTwoo** [comment]
Are the translations for Spanish and Portuguese correct?

**Comment by carlosabcs** [comment]
We have a tool called Problem Finder in the platform so that this translation could confuse users. I think it would be better to use:
"View problem collections" (en)  -> "Ver colecciones de problemas" (es)
"View problem list" (en)  -> "Ver lista de problemas" (es)

@pabo99 please correct me if I'm wrong.

**Comment by YannyTwoo** [comment]
Ok, changed translations should be good.

**Comment by YannyTwoo** [comment]
@pabo99 are there any changes I should make?

File: frontend/www/js/omegaup/components/common/NavbarItems.vue [hunk 1]
```diff
@@ -102,12 +102,18 @@
...
         <div class="dropdown-menu">
           <slot name="problems-items">
+            <a class="dropdown-item" href="/problem/" data-nav-problems-list>{{
+              T.navViewProblemsAll
+            }}</a>
             <a
               class="dropdown-item"
               href="/problem/collection/"
               data-nav-problems-collection
               >{{ T.navViewProblems }}</a
```
Comment (heduenas) [inline frontend/www/js/omegaup/components/common/NavbarItems.vue:112]: Can you rename this option to be "Browse Problems" and the new option "Problem List"?
Also, put the "Browse Problems" option first and the "Problem List" option second.

Comment (YannyTwoo) [reply frontend/www/js/omegaup/components/common/NavbarItems.vue:112]: Ok got it. Naming the new link to '/problem' as "Problem List"

**Review by heduenas — APPROVED** [review]

**Pull request merged**